ingest/scripts/group_segments.py

```diff
@@ -48,6 +61,7 @@ class Config:
     segmented: bool
 
 
+# TODO: Better name needed, what is special about these fields?
```

Comment:
segment is by definition different across segments, so this is unnecessary to make configurable (hard coded special field)